### ISSUE TITLE
[Request] Assign the project new maintainers

### ISSUE THREAD

**Issue by fehbari** [Ashald/EnvFile#155]
Given the current state of the plugin, with the installation reported [broken for months](https://github.com/ashald/EnvFile/issues/140), a [critical fix](https://github.com/ashald/EnvFile/pull/147) waiting weeks for a merge, and the author claiming multiple times that he no longer has time for open-source activity, it would be in the best interest of the community that a trusty 2nd maintainer (or group of maintainers) be assigned merge rights in the repo, and publishing rights in the JetBrains marketplace.

To be clear I sincerely appreciate the great effort made by @ashald in this plugin. However, there's potentially thousands of devs relying on this plugin in their workflow daily, so timely critical fixes should be a priority. I'm sure the community would be eager to help (as seen by the workarounds shared in issues opened in the past couple months), we just need the rights to maintain the project when the author is not around.

Thoughts? @ashald Please take this into consideration. Thank you.

**Comment by joaonc** [comment]
Agreed and hope @ashald does add someone to help reviewing and merging PRs.

I think it would help to have people suggestions here..   anyone volunteering to be that person?  (wink, wink @tsalzinger who fixed https://github.com/ashald/EnvFile/issues/140).

Another alternative would be to clone this repo and call it **EnvFile2**, which is a practice done often (again, volunteers are needed).

I use this plugin in PyCharm and it became indispensible in my current flow.  In fact, this plugin is so cool that I don't see why JetBrains doesn't include this as standard functionality!  Awesome job @ashald

**Comment by JSorella** [comment]
I am able to help if it's necessary!

**Comment by muhammednagy** [comment]
am able to help as well.

**Comment by Thorbenl** [comment]
I can help too!

**Comment by eltonsandre** [comment]
I can help too!

**Comment by DSEyers** [comment]
Or create new Repo with a copy of the code on the basis its not copy protected.

**Comment by ashald** [comment]
TL;DR: I have no intention of doing so. Though feel free to fork it, it's OSS.

Now, some details for those who are interested.
First of all, this particular repo doesn't mean anything. Anyone can fork it, and many already did so.
What really matters is ability to publish the plugin to JetBrains Marketplace. And I was doing so under my personal account so far.
I have no intention of giving access to anyone to my personal account on JetBrains marketplace, neither I have capacity to look into setting up an organization, and then maintaining it. Similarly, for this repo - it'a repo under my personal account, and I have no intention giving anyone access to manage it. Neither I have capacity to setup an org and transfer this repo there.
Neither I trust random strangers not to trash this repo if I just "give it away."

I don't know what I want to do with this repo long-term, but for now I want it to sit on my profile as is, serving as a landing page for the plugin I publish to JetBrains Marketplace under my name. Maybe I will work on it more in future once I have more free time, or maybe it will stick around as low-maintenance thing as it's right now. Last time I had high spirit for working more on this plugin JetBrains refused to give me license to their Pro versions of IDEs [that I needed to test integrations] citing low activity on the plugin [duh, and I was about to make it high activity lol].

Once again, as I mentioned above, it's OSS so any of you can fork this repo and publish it under your name, or make an organization out of a fork, build an amazing community around it, and publish it under organization account. Whatever you want to do, as long as it's aligned with the license this code is released under (MIT as of today).

Frankly speaking, I just hope JetBrains implements something similar to EnvFile in their IDEs and that would be the end of the story.

P.S.: I'm impressed by the amount of support offered by the community once the plugin broke, though I wish it was equally demonstrated when plugin just works as well.

**Comment by 1oglop1** [comment]
Thank you @ashald This plugin is an essential part of IntelliJ and JB refuses to integrate it for 10 years! 
It's a real shame they did not want to provide you with a license!
There are now very few things that IntelliJ can do and VSCode cannot.
